board/safety/safety_toyota.h

```diff
@@ -29,14 +35,59 @@ const int32_t RT_INTERVAL = 250000;    // 250ms between real time checks
 const int16_t MAX_ACCEL = 1500;        // 1.5 m/s2
 const int16_t MIN_ACCEL = -3000;       // 3.0 m/s2
 
+int cruise_engaged_last = 0;           // cruise state
+int ipas_state = 1;                    // 1 disabled, 3 executing angle control, 5 override
+int angle_control = 0;                 // 1 if direct angle control packets are seen
+double speed = 0.;
+
+// track the torque measured for limiting
+int16_t torque_meas[3] = {0, 0, 0};    // last 3 motor torques produced by the eps
```

Comment:
This should be a struct, and a function should update it. No repeating code three times